test/src/unit-cppapi-enumerations.cc

```diff
@@ -313,6 +325,67 @@ TEST_CASE_METHOD(
   REQUIRE(enmr_name2.has_value() == false);
 }
 
+TEST_CASE_METHOD(
+    CPPEnumerationFx,
+    "CPP: Enumerations From Disk - ArraySchema::get_enumeration",
+    "[enumeration][array-schema-get-enumeration]") {
```

Comment:
please also add the `[rest]` tag so that this runs on REST-CI as well and verifies the behavior is identical/right for `tiledb://` URIs.

Reply:
5d5c3b